scenes/game/game.gd

```diff
@@ -1480,12 +1488,20 @@ func _on_exceed_event(event):
 func _on_exceed_revert_event(event):
 	var player = event['event_player']
 	if player == Enums.PlayerId.PlayerId_Player:
-		$PlayerCharacter.set_exceed(false)
+		if 'exceed_animation' in player_deck:
```

Comment:
this would be pretty cool with leo to use

Reply:
I ended up making a separate field (wide_animation) so it happens after the position reset, but I'll keep this around just in case